zerver/tests/test_auth_backends.py

```diff
@@ -6064,14 +6091,15 @@ def test_login_success_when_user_does_not_exist_with_split_full_name_mapping(sel
             LDAP_APPEND_DOMAIN="zulip.com",
             AUTH_LDAP_USER_ATTR_MAP={"first_name": "sn", "last_name": "cn"},
         ):
+            splitname_email = self.nonreg_email("newuser").replace("@", "_splitname@")
```

Comment:
I think this hunk isn't helpful -- it's nice that the previous code was explicit.

It might be better to just have `self.nonreg_email("newuser_splitname")` work.